python/tvm/contrib/hexagon/build.py

```diff
@@ -689,18 +556,19 @@ def start_server(self):
             self._copy_to_remote(lib_dir / item, self._workspace / item)
         # Copy libc++ from the toolchain to the workspace
         self._copy_libcxx(self._workspace)
-        self._device_key = self.HEXAGON_REMOTE_DEVICE_KEY + "." + str(os.getpid())
+        self._rpc_info["device_key"] = HEXAGON_REMOTE_DEVICE_KEY + "." + str(os.getpid())
```

Comment:
On simulator, each test initializes an RPC server. `os.getpid()` is used to distinguished between those server, otherwise one test could connect to an RPC server (by mistake) that was created before but the shutdown process for it was already started.

Reply:
Thank you, and that makes sense!